unittests/test_cli.py

```diff
@@ -71,15 +71,16 @@ def setUp(self):
         self.prefix = tempfile.mkdtemp(dir='unittests')
         self.system = 'generic:login'
         self.checkpath = ['unittests/resources/hellocheck.py']
-        self.environs  = ['builtin-gcc']
```

Comment:
This formatting is also allowed by our coding style. I'm not against the strict PEP8 on that, but I find the alignment nice if you only need to pad with at maximum so many spaces as the length of the following operator (here one).